cromshell

```diff
@@ -1145,6 +1163,8 @@ function list()
     # Restore cursor position:
     tput rc
   fi
+  pretty_ugly=$(echo $UNREACHABLE_SERVER | sed 's/ /\n/g')
```

Comment:
This should be in an if block so it doesn't print if the unreachable server list is empty.

Reply:
done